### PR TITLE
minor english corrections

### PR DESCRIPTION
**Pull request by CoriPegliasco**

### CODE REVIEW

File: src/py_eddy_tracker/dataset/grid.py [hunk 1]
```diff
@@ -1275,7 +1275,7 @@ def get_step_in_km(self, lat, wave_length):
         min_wave_length = max(step_x_km, step_y_km) * 2
         if wave_length < min_wave_length:
             logger.error(
-                "Wave_length too short for resolution, must be > %d km",
+                "Wavelength too short for resolution, must be > %d km",
```
Comment (AntSimi) [inline src/py_eddy_tracker/dataset/grid.py:1278]: In this case we talked about variable which are called "wave_length", i don't know the best soltuion

File: src/py_eddy_tracker/dataset/grid.py [hunk 2]
```diff
@@ -1315,7 +1315,7 @@ def finalize_kernel(self, kernel, order, half_x_pt, half_y_pt):
 
     def kernel_lanczos(self, lat, wave_length, order=1):
         """Not really operational
-        wave_length in km
+        wavelength in km
```
Comment (AntSimi) [inline src/py_eddy_tracker/dataset/grid.py:1318]: n this case we talked about variable which are called "wave_length", i don't know the best soltuion

File: src/py_eddy_tracker/dataset/grid.py [hunk 3]
```diff
@@ -1327,7 +1327,7 @@ def kernel_lanczos(self, lat, wave_length, order=1):
         return self.finalize_kernel(kernel, order, half_x_pt, half_y_pt)
 
     def kernel_bessel(self, lat, wave_length, order=1):
-        """wave_length in km
+        """wavelength in km
```
Comment (AntSimi) [inline src/py_eddy_tracker/dataset/grid.py:1330]: n this case we talked about variable which are called "wave_length", i don't know the best soltuion, maybe  add docstring for argument like :
:param float wave_length: in km

File: src/py_eddy_tracker/observations/network.py [hunk 4]
```diff
@@ -255,6 +257,12 @@ def correct_close_events(self, nb_days_max=20):
         :param float nb_days_max: maximum time to search for splitting-merging event
         """
 
+        # FIXME : we want to change
+        # segment A splits from segment B, then x days after segment B merges with A
+        # to
```
Comment (AntSimi) [inline src/py_eddy_tracker/observations/network.py:262]: maybe create a project note or an issue for improvement, about that

File: src/py_eddy_tracker/observations/network.py [hunk 5]
```diff
@@ -1197,14 +1205,17 @@ def fully_connected(self):
         return self.tag_segment().shape[0] == 1
 
     def remove_trash(self):
+        """
+        Remove the lonely eddies (only 1 obs in segment, associated segment number is 0)"""
```
Comment (AntSimi) [inline src/py_eddy_tracker/observations/network.py:1209]: i don't think black allow """" on same line?

File: src/py_eddy_tracker/observations/observation.py [hunk 6]
```diff
@@ -885,7 +885,7 @@ def load_from_netcdf(
         :param bool raw_data: If true load data without apply scale_factor and add_offset
         :param None,list(str) remove_vars: List of variable name which will be not loaded
         :param None,list(str) include_vars: If defined only this variable will be loaded
-        :param None,dict indexs: Indexs to laad only a slice of data
+        :param None,dict indexs: Indexs to laod only a slice of data
```
Comment (AntSimi) [inline src/py_eddy_tracker/observations/observation.py:888]: Still a typo :)

**Review by AntSimi — CHANGES_REQUESTED** [review]

File: src/py_eddy_tracker/generic.py [hunk 7]
```diff
@@ -57,6 +57,7 @@ def reverse_index(index, nb):
     :return: mask of value selected
     :rtype: array
     """
+
```
Comment (AntSimi) [inline src/py_eddy_tracker/generic.py:60]: why new blank line

File: src/py_eddy_tracker/generic.py [hunk 8]
```diff
@@ -76,6 +77,7 @@ def build_index(groups):
     >>> build_index(array((1, 1, 3, 4, 4)))
     (array([0, 2, 2, 3]), array([2, 2, 3, 5]), 1)
     """
+
```
Comment (AntSimi) [inline src/py_eddy_tracker/generic.py:80]: why new blank line

Comment (CoriPegliasco) [reply src/py_eddy_tracker/generic.py:80]: Sometimes blanck lines appear after the docstring, sometimes not, I'm confused

Comment (CoriPegliasco) [reply src/py_eddy_tracker/generic.py:80]: (in groups.py for example)

**Review by AntSimi — APPROVED** [review]

**Pull request merged**